cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -126,7 +127,17 @@ void process_batch(std::vector<IndexDescriptor>& query_index_descriptors,
                 // print overlaps
                 if (print)
                 {
-                    print_paf(overlaps, cigars, *query_parser, *target_parser, KMER_SIZE, print_mutex);
+                    if (format == OutputFormat::PAF)
```

Comment:
This `if` could also be put in an `#ifdef GW_BUILD_HTSLIB`

Reply:
I'm a little one the fence about this one since that field will always be populated to the default  value of OutputFormat::PAF, so it's relatively harmless. That said, I changed it